bc_obps/bc_obps/settings.py

```diff
@@ -47,6 +47,7 @@
 ]
 
 MIDDLEWARE = [
+    'registration.kubernetesMiddleware.KubernetesHealthCheckMiddleware',
```

Comment:
We might also need to change this if we decide to change the name of the module and add a middleware folder.